kodilla-cucumber/src/test/java/seasonchecker/IsFizzOrBuzzSteps.java

```diff
@@ -3,56 +3,62 @@
 import io.cucumber.java8.En;
 import org.junit.Assert;
 
-import java.time.MonthDay;
-
 public class IsFizzOrBuzzSteps implements En {
-    private MonthDay today;
+    private int number;
```

Comment:
public class IsFizzOrBuzzSteps implements En {

    private int number;
    private String message;

    public IsFizzOrBuzzSteps() {
        Given("I have a {int}", (Integer numberInput) -> {
            this.number = numberInput;
        });

        When("Checking if number is divisible by (\\d+) and (\\d+)", (Integer arg0, Integer arg1) -> {
            if (number % 3 == 0 && number % 5 == 0) {
                this.message = "FizzBuzz";
            } else if (number % 3 == 0) {
                this.message = "Fizz";
            } else if (number % 5 == 0) {
                this.message = "Buzz";
            } else {
                this.message = "None";
            }
        });

        Then("I should be returned {string}", (String string) -> {
            assertEquals(string, message);
        });
    }
}

Reply:
Sprobuj tak